controllers/grid/users/reviewer/form/ReviewerForm.inc.php

```diff
@@ -241,12 +242,16 @@ function fetch($request) {
 			'reviewDueDate' => __('reviewer.submission.reviewDueDate'),
 			'submissionReviewUrl' => __('common.url'),
 			'reviewerUserName' => __('user.username'),
+			'contextName' => $context->getLocalizedName(),
```

Comment:
This one is apparently already in MailTemplate class, s. the function assignParams https://github.com/pkp/pkp-lib/blob/master/classes/mail/MailTemplate.inc.php#L144